src/js/components/presentational/1-atoms/date-term/DateTerm.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+import React from "react"
+import { format as formatFunc, differenceInCalendarDays } from 'date-fns'
+
+export const getRelativeDay = (date, relativeDate = new Date()) => {
+  const isRelativeDateANumber = (relativeDate instanceof Date)
+
+  const dateDifference = isRelativeDateANumber ? differenceInCalendarDays(date, relativeDate) : '0'
+
+  let output = `${dateDifference} days left`
+
```

Comment:
As a further optimisation, I would suggest having a function that calculates the day difference that does all the error checking and isNan() stuff. And a different function to display text based on that day difference value.

We then could swap out the day difference function for something in date-fns npm package (if something like that exists)